verifier/src/main/java/com/aws/jverify/verifier/compiler/JavaToDafnyCompiler.java

```diff
@@ -75,14 +77,28 @@ public NameCompiler getNameCompiler() {
     }
     
     public @Nullable FilesContainer analyzeJavaCode(VerifierOptions options, List<JavaFileObject> files) {
-        var parsed = new JavaFrontEnd(this).parseResolveAndDesugarJava(options, files);
-        if (parsed == null) {
+        Iterable<JCTree.JCCompilationUnit> parsedIterable = new JavaFrontEnd(this).parseResolveAndDesugarJava(options, files);
+        if (parsedIterable == null) {
             return new FilesContainer(List.of());
         }
+        
+        List<JCTree.JCCompilationUnit> parsed = new ArrayList<>();
+        for (var file : parsedIterable) {
+            parsed.add(file);
+        }
```

Comment:
```suggestion
        parsedIterable.forEachRemaining(parsed::add);
```

Or AFAICT we can make parseResolveAndDesugarJava return a Collection instead of just an Iterable and then you can just say `new ArrayList(parsed)`.